src/deployment.jl

```diff
@@ -1,41 +1,57 @@
 """
-    deployment_area(n_corals::Int64, max_n_corals::Int64, density::Float64, target_areas::Vector{Float64})::Tuple{Float64,Float64}
+    version_check()::Vector{Int64}
+
+Check RME version
+"""
+function version_check()::Vector{Int64}
+    rme_ver = @RME version()::Cstring
+    return parse.(Int64, split(rme_ver, '.'))
+end
```

Comment:
I have an issue with the function name in that it doesn't perform a "check".

To me, it should be `rme_version()` or similar as it simply returns the version number (albeit separated).
Simply `version_check` can be confusing. It's unclear at first glance: What does it mean? Version of the Julia package?

I'm also hesitant to accept it returning a vector of integers. Version numbers should be treated as IDs firstly.

I also dislike how this is being used. Remember that if you're hardcoding index values, then it's usually a smell telling you there's a better way.

In this case, I would prefer:

1. a named tuple be returned: `(major=1, minor=0, patch=28)` (following [SemVer](https://semver.org/))
2. the function name be changed to `rme_version_info()`